manual-workflows/start.sh

```diff
@@ -132,18 +148,35 @@ EOF
     exit 1
 fi
 
+MACHINE_TYPE=${MACHINE_TYPE:-"e2-standard-2"}
+
+param=(
+  --project $PROJECT
+  --image-family debian-11
+  --image-project debian-cloud
+  --zone $ZONE
+  --service-account=$SERVER_ACCOUNT
+  --scopes=cloud-platform
+  --network=$NETWORK
+  --subnet=$SUBNET
+  --metadata=cromwell-version=71
+  --metadata-from-file=startup-script=$SRC_DIR/server_startup.py,cromwell-conf=$CROMWELL_CONF,helpers-sh=$SRC_DIR/helpers.sh,cromwell-service=$SRC_DIR/cromwell.service,workflow-options=$WORKFLOW_OPTIONS,persist-artifacts=$SRC_DIR/../scripts/persist_artifacts.py
+)
+
+# optionally add either machine type when neither --custom-memory
+# nor --custom-cpu are specified
+if [ -z "$MEM" ] || [ -z "$CPU" ]; then
+  echo "$MEM BLAAAAH $CPU"
```

Comment:
I like `set -x` for debugging shell scripts! (`echo` works too 🙂--just a reminder to remove this line before we merge it in.)